client/src/components/Ticket.js

```diff
@@ -0,0 +1,110 @@
+import React, { useState, useEffect } from 'react';
+import './Ticket.css';
+import ReactCardFlip from 'react-card-flip'; // comopnent that will flip the ticket
+
+function Ticket(props) {
+  const [isFlipped, setIsFlipped] = useState(true); // state for flipping animation
+  const [contentClass, setContentClass] = useState('content'); // state for changing the content class name
+  const [display, setDisplay] = useState('flex'); // state for hidding and displaying the ticket
```

Comment:
Better use a boolean then a CSS property as a state.

Reply:
Can you explain the benfits of boolean over css property? Nir showed this option in a lesson.